pkg/manifests/config.go

```diff
@@ -66,6 +66,21 @@ func (c Config) IsStorageConfigured() bool {
 	return prometheusK8sConfig.VolumeClaimTemplate != nil
 }
 
+func (c Config) HasInconsistentAlertmanagerConfigurations() bool {
+	if c.ClusterMonitoringConfiguration == nil && c.UserWorkloadConfiguration == nil {
+		return false
+	}
+
+	alertmanagerMainConfig := c.ClusterMonitoringConfiguration.AlertmanagerMainConfig
```

Comment:
nit: `main` IMHO is verbose enough :) https://github.com/golang/go/wiki/CodeReviewComments#variable-names 

Again, it is just a nitpick

Reply:
No, when reviewing my code feel free to always nitpick, I also agree it look better!